apps/flux-system/base/image-automation-components.yaml

```diff
@@ -1995,7 +2037,7 @@ spec:
     spec:
       containers:
       - args:
-        - --events-addr=http://notification-controller/
```

Comment:
is this intentionally changing?

Reply:
Mentioned this to Praveen earlier. I checked sds and not set there. Do you know if it should be? Was testing without it.